README.md

```diff
@@ -214,7 +214,7 @@ service account token that has read-only access to the Kubernetes cluster.
 
 #### Kubernetes Deployment
 
-To deploy this project, you can simply run `kubectl apply -f kubernetes` and a
+To deploy this project, you can simply run `kubectl apply -f kubernetes/standard` and a
```

Comment:
standard? Hmm we can come up with a better name I think here.

Reply:
Any suggestions? I haven't been able to come up with anything better, and this is the standard way to deploy kube-state-metrics.

Reply:
We said we want to also since we moving it, rename the directory to `examples`.